scripts/engineering/aggregate_phase44d_carry_aware_experimental_family_matrix.py

```
@@ -0,0 +1,356 @@
+            raise SystemExit(f"duplicate row key in {source}: {key}")
+        row_map[key] = row
+    return row_map
+
+
+def round3(value: float) -> float:
+    return round(value, 3)
+
+
+def main() -> None:
+    args = parse_args()
+    first_blocked_overrides = {family: value for family, value in args.first_blocked_step}
```

Ruff (0.15.11)

[warning] Unnecessary dict comprehension (rewrite using `dict()`)

Rewrite using `dict()`

(C416)